Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -102,13 +191,37 @@ lemma restrictScalars_eq :
 
 end normalClosure
 
+variable {F K L}
+
+open Cardinal in
+/-- An extension `L/F` in which every minimal polynomial of `K/F` splits is maximal with respect
+  to `F`-embeddings of `K`, in the sense that `K →ₐ[F] L` achieves maximal cardinality. -/
+theorem Algebra.IsAlgebraic.cardinal_mk_algHom_le_of_splits (alg : IsAlgebraic F K)
+    (h : ∀ x : K, (minpoly F x).Splits (algebraMap F L)) (L' : Type v) [Field L'] [Algebra F L'] :
+    lift.{u} #(K →ₐ[F] L') ≤ lift.{v} #(K →ₐ[F] L) := by
```

Comment:
It seems like the proof constructs an inclusion. Would it make more sense to just state that there is an inclusion rather than state this result in terms of cardinals?

Reply:
switched to embedding